README.md

```diff
@@ -171,6 +171,8 @@ The rest of these Environment Variables can be set to match AWS Lambda's environ
 * `AWS_LAMBDA_FUNCTION_VERSION`
 * `AWS_LAMBDA_FUNCTION_NAME`
 * `AWS_LAMBDA_FUNCTION_MEMORY_SIZE`
+* `AWS_LAMBDA_RIE_DISABLE_EXTENSIONS` - Disables AWS Lambda extensions when working with SAM; setting this environment variable to any value other than `FALSE` will disable extensions.
```

Comment:
Small suggested change to explicitly mention that it's RIE only (because the whole section is supposed to be related to Lambda environment variables, so it can be confusing), and to not mention SAM (because this also works for customers using RIE directly):

```md
* `AWS_LAMBDA_RIE_DISABLE_EXTENSIONS` - (RIE only) Disables AWS Lambda extensions when working locally. Setting this environment variable to any value other than `FALSE` will disable extensions.
```

Reply:
Done